docs/docs/developers/contracts/writing_contracts/storage/notes.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Using Notes
+---
+
+Some optionally background resources on notes can be seen here:
+- [High level network architecture](../../../../learn/about_aztec/technical_overview#high-level-network-architecture), specifically the Private Execution Environment
+- [Transaction lifecycle (simple diagram)](../../../../learn/concepts/transactions#simple-example-of-the-private-transaction-lifecycle)
+- [Public and Private state](../../../../learn/concepts/hybrid_state/main)
+
+Outside of this there are many more details to discover that will be skipped over for now:
+- Private and public contexts
+- Encryption keys and events
+- Oracle's role in using notes
+- Value Serialization/Deserialization
+
+This explainer will go through some examples that touch on what has been covered [here](../../../../learn/concepts/storage/notes).
```

Comment:
Can you give a bullet point list of what this explainer goes through (& put this before what is omitted)?